src/index.js

```diff
@@ -205,9 +212,14 @@ const SwipeableViews = React.createClass({
       translate: spring(index * 100, springConfig),
     };
 
+    const touchEvents = !this.props.disabled ? {
+      onTouchStart: this.handleTouchStart,
```

Comment:
Can we use the spread to get the disabled property?
I plan to pass unused properties to the root div, so I need it.
I also think that negation are harder to reason about.